app/models/hearings/forms/base_hearing_update_form.rb

```diff
@@ -105,6 +105,19 @@ def should_create_or_update_virtual_hearing?
     )
   end
 
+  # This is an existing hearing that is being converted to virtual if:
+  # - There is no virtual hearing
+  # - This hearing exists with a request type that is not virtual ("R")
+  def conversion_to_virtual?
```

Comment:
I think i prefer wither `converting_to_virtual?` or `convert_to_virtual?`
```suggestion
  def converting_to_virtual?
```

Reply:
Good point. Done.